junit-platform-runner/src/main/java/org/junit/platform/runner/JUnitPlatform.java

```diff
@@ -255,6 +266,19 @@ private void addExcludedEnginesFilter(LauncherDiscoveryRequestBuilder requestBui
 		return patterns;
 	}
 
+	private String[] getExcludeClassNamePatterns() {
+		String[] patterns = getValueFromAnnotation(ExcludeClassNamePatterns.class, ExcludeClassNamePatterns::value,
+			new String[0]);
+
+		if (patterns.length == 0) {
+			return new String[] {};
```

Comment:
True enough, thanks for pointing out. On second thoughts I decided to drop the check for zero length altogether. The following code is robust with respect to zero length arrays and the resulting code is simpler.